### DISCUSSION TITLE
[Dotnet8] New duplicate image errors

### DISCUSSION THREAD

**Discussion by haavamoa** [dotnet/maui#17433] [Q&A] [answered]
Hi, when we upgraded to Dotnet8 RC we got the new duplicate image errors. We are creating two libraries where both include MAUIImage for the consuming app. One library refer thee other, so we are suspecting that this raises the issue.

Due to internal policies in my company I am unable to share the code, and I am unable to reproduce it in a slim project to share here on Github.

Before spending more time on this issue: Is there any way of opting out of this behavior? Chaning from Include to Update does not solve it. Is there a ‘accept-duplicates’ flag or something we can set?

**Comment by Vetle444** [comment]
Bump!

**Comment by Vetle444** [comment, marked as answer]
ResizetizerErrorOnDuplicateOutputFilename is the property you have to set to false in .csproj to disable the error!

**Comment by haavamoa** [comment]
Sweet, that works! Thanks!
https://github.com/dotnet/maui/blob/d9eedd6acd221e971e97cf6fc8af3d4271cf5440/src/SingleProject/Resizetizer/src/nuget/buildTransitive/Microsoft.Maui.Resizetizer.After.targets#L93
